config/homebin/vagrant_up

```diff
@@ -46,5 +47,8 @@ else
 	echo -e "${RED}   ▄▄▀▄${TEARS}░░░░░${TEARS}▄▀▄▄    Check the logs and reprovision."
 	echo -e "${RED}                      If that fails, go to the VVV Slack or open an issue"
 	echo -e "${RED}                      on GitHub. Logs are in the logs/provisioners folder."
+	echo -e "${RED}"
+	echo -e "${RED}   Logs relevant to this:"
+	ls -A /vagrant/failed_provisioners/ | xargs -I "LOGFILE" echo -e "   * ${date_time}/LOGFILE.log"
```

Comment:
It isn't clear for me this line

Reply:
This list the files that signal the failed provisioners and pipes the output to xargs, which takes each line piped to it and run a command.

I'm this case it runs an echo, and replaces LOGFILE with the file being piped to it.

This would basically print the logs to look for details of the failure.

Reply:
Can we get that added as a one line comment just so we know what it does

Reply:
can we add a 1liner comment explaining this?

Reply:
Sure, done